src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -0,0 +1,377 @@
+        leave={{ opacity: 0, enterProgress: 1 }}
+        config={springs.smooth}
+      >
+        {show =>
+          show &&
+          /* eslint-disable react/prop-types */
+          (({ opacity, enterProgress }) => (
+            <AnimatedDiv
+              style={{
+                opacity,
+                transform: enterProgress.interpolate(
+                  v => `
+                    scale3d(${1 - (1 - v) * 0.03}, ${1 - (1 - v) * 0.03}, 1)
+                  `
+                ),
+              }}
+            >
+              <IconQuestion />
+            </AnimatedDiv>
+          ))
+        }
+      </Transition>
+    </RoundButton>
+  )
+}
+
+ToggleDialogueButton.propTypes = {
+  open: PropTypes.bool.isRequired,
+  onToggle: PropTypes.func.isRequired,
+}
+
+const OptInDialogue = React.memo(({ onClose, onOptIn, ...styles }) => {
+  return (
+    <animated.div {...styles}>
+      <Wrapper>
+        <Header>
+          <h2
+            css={`
+              font-size: 18px;
+              line-height: 22px;
+            `}
+          >
+            How can we help?
+          </h2>
+          <img
+            css={`
+              margin-top: ${2.5 * GU}px;
+            `}
+            src={logo}
+            alt="Aragon logo"
+          />
+        </Header>
+        <main
+          css={`
+            display: flex;
+            flex: 1;
+            padding: ${2 * GU}px;
+            overflow: auto;
+          `}
+        >
+          <Body>
+            <h3
+              css={`
+                font-weight: bold;
+                font-size: 18px;
+              `}
+            >
+              We need your consent.
+            </h3>
+            <p>
+              We want to assist you in using the product, providing help, and
+              answers to common questions.
+            </p>
+            <p>
+              For that, we use a third-party system called{' '}
+              <StyledSafeLink href="https://www.helpscout.com/">
+                Help Scout
+              </StyledSafeLink>
+              . If you opt-in, we will load their program onto Aragon.
+              <br />
+              Help Scout is a{' '}
+              <StyledSafeLink href="https://bcorporation.net/directory/help-scout">
+                Public Benefit Corp
+              </StyledSafeLink>
+              .
+            </p>
+            <Button
+              mode="strong"
+              wide
+              onClick={onOptIn}
+              css={`
+                margin-top: ${1 * GU}px;
+                font-size: 15px;
+              `}
+            >
+              Yes, I'd like help
+            </Button>
+          </Body>
+        </main>
+      </Wrapper>
+    </animated.div>
+  )
+})
+
+OptInDialogue.propTypes = {
+  onClose: PropTypes.func.isRequired,
+  onOptIn: PropTypes.func.isRequired,
+}
+
+const Wrapper = styled.aside`
+  background: #fff;
+  border: 1px solid rgba(209, 209, 209, 0.5);
```

Comment:
This semi-transparent border combined with the shadow makes the shape look a bit blurry, maybe we could remove it since we already have a shadow? /cc @dizzypaty